src/test/FuzzerImpl.cpp

```diff
@@ -503,17 +514,27 @@ isBadOverlayFuzzerInput(StellarMessage const& m)
 
 static void
 attemptToApplyOps(LedgerTxn& ltx, PublicKey const& sourceAccount,
-                  xdr::xvector<Operation> const& ops, Application& app)
+                  xdr::xvector<Operation> const& ops, Application& app,
+                  bool const& throwIfTxFails = true)
```

Comment:
pass by value.. this is a bool

Reply:
not sure you need this parameter though as `attemptToApplyOps` is only called when performing setup and we always want to fail?

Reply:
> pass by value.. this is a bool

Fixed.

> not sure you need this parameter though as attemptToApplyOps is only called when performing setup and we always want to fail?

It's also called by `TransactionFuzzer::inject()`, which is what we call when running fuzz tests, isn't it?